### PR TITLE
CNV-9040 Supported DV List

### PR DESCRIPTION
**Pull request by bgaydosrh**
First draft of module for [CNV-9040](https://issues.redhat.com/browse/CNV-9040) (https://issues.redhat.com/browse/CNV-9040) for code review.

Questions:

1. I have used "volume" here to be as generic as possible, on the chance that the user is not uploading to a PVC. Should I say "datavolume"?
2. What does the user do if they want to create a boot source for an OS we don't list here? Contact support?
3. Should we say only the OSs listed in the table are supported?

See the Netify preview build and navigate to Virtual Machines > Virtual Disks > Creating and Using Boot Sources to review this new module. I have added a prereq to the assembly, as well.

Tagging @gouyang for QE Review
Tagging @aglitke for Code Review (but Adam, please send it to whomever is best to review this).

Thanks
Bob

### CODE REVIEW

File: modules/virt-using-supported-boot-source-names.adoc [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Use the following table to determine the name of the volume that you define as a boot source.
+
+[cols="2",options="header"]
+|===
+|To create a boot source for this OS:
+|Name the volume:
+
+|Fedora
+|fedora32
```
Comment (gouyang) [inline modules/virt-using-supported-boot-source-names.adoc:20]: the dv name of Fedora is 'fedora'.

File: modules/virt-using-supported-boot-source-names.adoc [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+|Ubuntu
+|ubuntu18
+
+|Windows 10
+|win10
+
+|Windows 2019 Server
+|win2k19
+
+|===
```
Comment (gouyang) [inline modules/virt-using-supported-boot-source-names.adoc:46]: Don't see dv name for RHEL6, Windows 2016 and Windows 2012 R2

File: modules/virt-using-supported-boot-source-names.adoc (hunk 1 above)
Comment (bgaydosrh) [reply modules/virt-using-supported-boot-source-names.adoc:18]: Fixed.

File: modules/virt-using-supported-boot-source-names.adoc (hunk 2 above)
Comment (bgaydosrh) [reply modules/virt-using-supported-boot-source-names.adoc:47]: Added.

**Comment by bgaydosrh** [comment]
Closing this PR. Replacing with

**Pull request closed without merging**